scripts/asr_language_modeling/ngram_lm/install_beamsearch_decoders.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
-# install Boost package
-sudo apt-get update
-sudo apt-get install swig build-essential libboost-all-dev cmake zlib1g-dev libbz2-dev liblzma-dev
+cd /workspace/nemo
```

Comment:
Let's not assume that all users have /workspace/nemo in this script.

Reply:
add $NEMO_PATH